services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -116,6 +118,8 @@
             continue
         node_details = await _get_node_details(node_key, node_version, app)
         log.debug("node %s:%s has schema:\n %s",node_key, node_version, pformat(node_details))
+        if node_details is None:
+            continue
         dag_adjacency_list = await _build_adjacency_list(node_uuid, node_details, node_inputs, pipeline_data, dag_adjacency_list, app)
```

Comment:
why do you need both changes? the first one should be sufficient no?

Reply:
without the second check the nodes-group would go into the adjacency_list and we want them away, right?

Reply:
then can't we have the same if checking for the nodes-group key? that would make easier to spot the next issue when there is a new type of node...

Reply:
mmm...